main/zoe/api/contract-support/price-authority.md

```diff
@@ -0,0 +1,141 @@
+# PriceAuthority Methods
+
+We rely on `PriceAuthority` oracles. A `PriceAuthority`
+gives reliable quotes for prices. The quotes might be based on broad surveys
+of prices across the ecosystem, or might come directly from an AMM (Automatic
+Market Maker) like our 
+AutoSwap. A `PriceAuthority` can either give a quote for the current price 
+across any pair of currencies it knows about, or can immediately return a 
+`Promise` resolved when a condition is true. For example, a price 
+crossing some threshold, or at a particular time. It can also provide a 
+price feed that updates with every price change.
+
+A `PriceQuote` represents a statement from the PriceAuthority as to the 
+current price level at a particular time. The significant content (prices 
+and time) is packaged in the amount, and repeated in the payment for veracity. 
+A `PriceQuote` is an amount-payment pair, where the amount is also the current 
+balance of the payment.
+
+It represents a statement from the `PriceAuthority` as to the current price
+level at a particular time. The significant content (prices and time) is 
+packaged in the amount, and repeated in the payment for veracity. 
```

Comment:
"veracity" is redundant with the previous paragraph. It there anything here that isn't already present above or below?